lzy/allocator/src/main/java/ai/lzy/allocator/util/KuberUtils.java

```diff
@@ -11,4 +11,12 @@ private KuberUtils() {
     public static boolean isResourceAlreadyExist(Exception e) {
         return e instanceof KubernetesClientException ex && ex.getCode() == HttpURLConnection.HTTP_CONFLICT;
     }
+
+    public static boolean isNotRetryable(KubernetesClientException e) {
+        var code = e.getCode();
+        if (code == 429) { // Too many requests
```

Comment:
`io.fabric8.kubernetes.client.dsl.internal.core.v1.PodOperationsImpl.HTTP_TOO_MANY_REQUESTS`  :)

Reply:
The word `internal` implies that we should use it. I would prefer using own constants

Reply:
Replaced it with constant